library-integration-slack/manifest.yml

```diff
@@ -3,7 +3,7 @@ applications:
 - name: library-integration-slack
   buildpack: https://github.com/cloudfoundry/java-buildpack.git#v4.15
   path: build/libs/library-integration-slack.jar
-  memory: 512M
+  memory: 1G
```

Comment:
`768M` should be enough to run the app without problems - and is 1/4 cheaper :)

Reply:
Good point, it is update in the cloud as well ;)